debug_gym/gym/envs/env.py

```diff
@@ -257,8 +257,24 @@ def _reset_env_state(self):
 
     def set_entrypoints(self, entrypoint: str, debug_entrypoint: str | None = None):
         if entrypoint:
-            self.entrypoint = entrypoint
-            debug_entrypoint = debug_entrypoint
+            is_python = "python " in entrypoint
+
+            self.entrypoint = self._prepare_entrypoint(entrypoint)
+
+            if debug_entrypoint is None:
+                debug_entrypoint = (
+                    entrypoint.replace("python ", "python -m pdb ", 1)
```

Comment:
Consider `entrypoint.replace("python ", "python -m pdb ")`, we might want to replay all instances of python like it used to. I'm not sure if that's valid scenario, but don't see a reason to change that.